### ISSUE TITLE
Yahoo Weather API Retired

### ISSUE THREAD

**Issue by deangwilcox** [felixhageloh/weather-widget#44]
This just in:
Important EOL Notice: As of Thursday, Jan. 3, 2019, the weather.yahooapis.com and query.yahooapis.com for Yahoo Weather API will be retired. 

To continue using our free Yahoo Weather APIs, use https://weather-ydn-yql.media.yahoo.com/forecastrss. Contact yahoo-weather-ydn-api@oath.com for credentials to onboard to this free Yahoo Weather API service.

https://developer.yahoo.com/weather/

Alternatives?

**Comment by caromun101** [comment]
Thanks, I love this widget!

**Comment by ChiefPoints** [comment]
Yup, I noticed the widget quit working a few days ago. This my favorite widget, hopefully it's not too hard to get it working again.

**Comment by Leptitmanu** [comment]
Hope that some good developer have some free time to make it work.
Thanks in advance for their effort

**Comment by Violinha** [comment]
My fav weather widget!

**Comment by samric2019** [comment]
Any plan on migrating this widget to Java Script while fixing the retired API?  I volunteer to beta test...

**Comment by Leptitmanu** [comment]
Hello
I've try but was unable to make the new Auth 2.0 working properly
I finally migrate on Wttr Forecast Widget
With some personalization I've reproduce the same look & field of the way I was
![capture d ecran 2019-02-10 a 12 10 43](https://user-images.githubusercontent.com/46262837/52532966-f42e5980-2d2c-11e9-8c70-52b8bd15435e.jpg)
 using the previous App

**Comment by coolajami** [comment]
that looks lovely, I suppose it can go in English? I believe it worth to put a pull request.

**Comment by Leptitmanu** [comment]
Yes you can choose the language
As said it is the Ubersicht widget Wttr Forecast
Once you download I can send you mu index.coffee to start with my "customization"

**Comment by coolajami** [comment]
Hi, I think ideally, it is better to share your index.cofee through a pull request if the code is similar with the current weather app. otherwise if you uploaded to your repo, we could download it from there. (I hope I got that right, I'm not a big github user. )

**Comment by whirka** [comment]
I would also love to see this as a PR

**Comment by Leptitmanu** [comment]
I publish my index.coffee in bkroggel/wttr in the pull request

I can comment if you have questions
